### DISCUSSION TITLE
Ideas: TS support, passing a path, specifying a template

### DISCUSSION THREAD

**Discussion by kaelig** [eduardconstantin/storybook-genie#13] [Ideas]
1. I'd love to be able to pass a path or a glob, rather than having to CD into a specific path.
2. Template: My team's stories.tsx files follow a particular template for consistency. I'd like to be able to point to a template file so that the output of the CLI always follows the same structure. 
3. TypeScript support (file contents and filenames extensions)

**Comment by joevaugh4n** [comment]
Following this! ❤️

**Comment by eduardconstantin** [comment]
I appreciate your suggestions! 
I will turn them into some issues, maybe the community would like to get involved. I plan to start working on some of them next week.
